scripts/generate_dump

```diff
@@ -974,11 +980,6 @@ save_symlink() {
     ${CMD_PREFIX}ln -s ../$src_dir/$file_basename $file_basename
     ${CMD_PREFIX}popd
 
-    if $do_tar_append; then
-        ($TAR $V -rf $TARFILE -C $DUMPDIR "$tar_path" \
```

Comment:
The change creates symlinks and avoids duplicate copy inside the tar. The final tar option has "-h" which will undo this change. Can we fix this?